documentation/book/cluster-operator.adoc

```diff
@@ -1030,9 +1030,9 @@ The default value of the `image` option is determined by the value of the `<<STR
 All other options have the same meaning as for the regular `KafkaConnect` deployment.
 
 Once the Kafka Connect S2I cluster is deployed, new plugins can be added by starting a new {OpenShiftName} build.
-Before starting the build, a directory with all the KafkaConnect plugins which should be added has to be created.
+Before starting the build, a directory with all the Kafka Connect plugins which should be added has to be created.
 The plugins and all their dependencies can be in a single directory or can be split into multiple subdirectories.
-For example:
+For example the following directory structure would be created in order to add the MySQL and Postgres CDC (change data capture) connectors provided by the Debezium[http://debezium.io/] project:
```

Comment:
Please add a comma after "example". The last is displaying the link to the Debezium project, instead of making Debezium a clickable link. Is this intentional?

Reply:
Instructions for links: http://strimzi.io/contributing/guide/#styling_links

Reply:
> Is this intentional?

Nope, got tripped up by Markdown vs. AsciiDoc link syntaxes.